desktop/backend/python/controller/controls/keyboard.py

```
@@ -0,0 +1,189 @@
+import time
+import pyautogui
+from typing import List, Union
```

Ruff (0.14.10)

`typing.List` is deprecated, use `list` instead

(UP035)